### DISCUSSION TITLE
Bug: Modal does not open if it is not executed in Livewire blade

### DISCUSSION THREAD

**Discussion by BubeDameKoenig** [livewire/flux#237] [Feature Requests]
If you do not insert the modal in a livewire blade, it does not open the modal.
It seems that there is still an error in the else.

<img width="1046" alt="Bildschirmfoto 2024-10-02 um 13 17 00" src="https://github.com/user-attachments/assets/42f362b2-1d72-41d2-9fb8-f29718ff8332">

**Comment by joshhanley** [comment]
@BubeDameKoenig please open an issue for a bug, not a feature request, so going to close this. Could you also make sure to share code so we can easily replicate the issue (not screenshots) with instructions on how to replicate it. Thanks!
